tools/agent-ctl/src/main.rs

```diff
@@ -218,7 +216,7 @@ fn real_main() -> Result<()> {
                 .long("log-level")
                 .short("l")
                 .help("specific log level")
-                .default_value(logging::slog_level_to_level_name(DEFAULT_LOG_LEVEL).unwrap())
+                .default_value(logging::slog_level_to_level_name(DEFAULT_LOG_LEVEL).map_err(|e| anyhow!(e))?)
```

Comment:
It's required because that function returns a `&'static str`. There might be a way to improve the code, but this is a trivial fix to a test tool so imho we don't need to labour this.